docs/reference/release-notes/7.9.asciidoc

```diff
@@ -22,3 +22,10 @@ Snapshot restore throttling::
   default that's used for `indices.recovery.max_bytes_per_sec`. This means
   that no behavioral change will be observed by clusters where the recovery
   and restore settings had not been adapted from the defaults. {es-pull}58658[#58658]
+
+Thread pool write queue size::
+* The WRITE thread pool default queue size (`thread_pool.write.size`) has been
+  increased from 200 to 10000. Additional memory-oriented back pressure has been
```

Comment:
Should we mention here that this new model now better supports indexing loads that consist of a few large bulks as well as many small bulks (from a larger number of clients).